internal/exportimport/handle_import.go

```diff
@@ -24,72 +24,77 @@ import (
 	"github.com/google/exposure-notifications-server/internal/exportimport/model"
 	"github.com/google/exposure-notifications-server/pkg/database"
 	"github.com/google/exposure-notifications-server/pkg/logging"
+	"github.com/hashicorp/go-multierror"
 
 	"go.opencensus.io/stats"
-	"go.opencensus.io/trace"
 )
 
-const lockPrefix = "import-lock-"
+const lockPrefix = "export-importer-worker-lock-"
```

Comment:
renaming these can cause things in flight to fail in weird ways

Reply:
@mikehelmick fixed